### PR TITLE
Superuman add

### PR DESCRIPTION
**Pull request by mhairimca**

### CODE REVIEW

File: docs/community/superUMAns.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+---
+title: Who are the SuperUMAns?
```
Comment (ChandlerDeKock) [inline docs/community/superUMAns.md:3]: What is the agreed writing of the word superUMA? 

SuperUMAn or superUMAn? Seen a few spots where you use this interchangeably. 

I guess it is the start of a sentence so it's not necessarily a proper noun - but maybe it should be?

**Review by ChandlerDeKock — COMMENTED** [review]
This is great. 

Added a few comments. A few formatting issues and also a few design discussion topics. 

Formatting issues are blocker
Design choices are not blockers but should be noted

File: docs/community/superUMAns.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+
```
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/superUMAns.md:1]: How do I fix this?!

I've fiddled about with it a bit and I thought I had, but it seems not.

File: docs/community/superUMAns.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+title: Who are the SuperUMAns?
+sidebar_label: SuperUMAns
+---
+
+
+## What is a SuperUMAns?
+
+SuperUMAns are the community's ninja troops.  They know how powerful UMA is, the kinds of things that UMA can be used for and they are determined to spread the word.
+
+*If you find yourself reading this, you probably already have your own inner superUMAn - you may just not know it yet...."
```
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/superUMAns.md:12]: argh - typo!

File: docs/community/superUMAns.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ - Brainstorm new use cases for existing UMA products
+ - Look for opportunities to collaborate with other communities
+ - Help teach people about what you can build using UMA
+ - Participate in UMA governance
+ - Support other communities on social media and in their local community chats.
+ -Support each other- we are a family
+ - Every member of our community has their own unique blend of superpowers and we value all of them
+
+## How do you become a superUMAn?
+All you have to do to become a superUMAn is show up and dive in! It really is that easy. If anything about UMA speaks to you, come join us on [Discord](https://discord.com/invite/jsb9XQJ) in the superUMAn lair. Bring all your questions!
```
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/superUMAns.md:46]: its the link from the main website so should be evergreen

File: docs/community/superUMAns.md (hunk 1 above)
Comment (mhairimca) [reply docs/community/superUMAns.md:3]: Yeah, capitalisation is the way to go!  Will change.

File: docs/community/superUMAns.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+ - Design awesome swag
+ - Hold weekly community calls
+ - Participate in online classes together
+ - Brainstorm new use cases for existing UMA products
+ - Look for opportunities to collaborate with other communities
+ - Help teach people about what you can build using UMA
+ - Participate in UMA governance
+ - Support other communities on social media and in their local community chats.
+ -Support each other- we are a family
+ - Every member of our community has their own unique blend of superpowers and we value all of them
```
[the remark this answers is not in the document]
Comment (mhairimca) [reply docs/community/superUMAns.md]: Yeah, its kindof an advert, its primary audience atm is the bankless folks that are coming over.

**Review by ChandlerDeKock — APPROVED** [review]
looks better

**Pull request merged**